drivers/storage/portworx/portworx_test.go

```diff
@@ -11701,6 +11701,105 @@ func TestGetKVDBMembers(t *testing.T) {
 
 }
 
+func TestGetKVDBMembersWithSecurityEnabled(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockServiceService := mock.NewMockPortworxServiceServer(mockCtrl)
+
+	sdkServerIP := "127.0.0.1"
+	sdkServerPort := 21883
+	mockSdk := mock.NewSdkServer(mock.SdkServers{
+		PortworxService: mockServiceService,
+	})
+
+	cluster := &corev1.StorageCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "px-cluster",
+			Namespace: "kube-test",
+		},
+		Status: corev1.StorageClusterStatus{
+			Phase: string(corev1.ClusterStateInit),
+		},
+		Spec: corev1.StorageClusterSpec{
+			Security: &corev1.SecuritySpec{
+				Enabled: true, // enable security
+				Auth: &corev1.AuthSpec{
+					GuestAccess: guestAccessTypePtr(corev1.GuestRoleDisabled), // guest access disabled
+				},
```

Comment:
It doesn't matter if guest access is enabled or not. We are always using token if security is enabled.

Reply:
That's correct, the test would still remain same. Added guest access to test with the requirement.